source/delayed_capture/index.html.md.erb

```diff
@@ -18,27 +18,36 @@ You cannot use delayed capture for:
 * payments made with [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/)
 * [recurring payments](/recurring_payments)
 
-To use this feature, include `"delayed_capture": true` in the body of a [Create new payment](/making_payments/#creating-a-payment) request.
+To use this feature, include `"delayed_capture": true` in the body of your [request to create a new payment](/making_payments/#creating-a-payment).
 
-The user experience matches the current payment flow. Once your user selects
```

Comment:
who are we referring to as your user ?

Reply:
I think this comment is on the old version of the page, but I've clarified it a bit either way by saying 'Your paying user'.